packages/hooks/src/hooks.spec.tsx

```diff
@@ -368,4 +369,13 @@ describe('packages/hooks', () => {
       expect(result.current.onChange).toBeDefined();
     });
   });
+
+  describe('useSsrCheck', () => {
+    it('should return true when server-side rendered and false after hydration', () => {
```

Comment:
nit: I _think_ we prefer `test()` vs `it()`. Not sure if there's any difference other than stylistic

Reply:
Oh good call out, totally missed this. Will update!